controller/backing_image_controller.go

```diff
@@ -212,14 +212,10 @@ func (bic *BackingImageController) syncBackingImage(key string) (err error) {
 			log.Info("Need to wait for all replicas stopping using this backing image before removing the finalizer")
 			return nil
 		}
-		cleanUpTmpFileFinished, err := bic.IsBackingImageDataSourceCleanup(backingImage)
-		if err != nil {
+		if _, err := bic.IsBackingImageDataSourceCleaned(backingImage); err != nil {
+			log.Info("Need to wait until backing image data source is cleaned before removing the finalizer")
```

Comment:
Suggest using warn because actually here we encountered an error, even though it will be returned to the caller.